Encrypting Current User and Local Storage (#695)
ParsePlatform/Docs: _includes/js/users.md

If you need to check if a `Parse.User` is authenticated, you can invoke the `authenticated` method. You do not need to check `authenticated` with `Parse.User` objects that are obtained via an authenticated method.

## Encrypting Current User

Often you want to be more careful with the user information stored in the browser, if this is your case try the encryption of the current user object. To enable this functionality, call `Parse.enableEncryptedUser()` and set your secret token.

```javascript

Parse.enableEncryptedUser();

> Parse.encryptedKey = 'my Secrey Key';

Reviewer: Quick nitpick, *secret*

Author: 😅 it's better, let me change it

Author: done 👌

If you need to check if a `Parse.User` is authenticated, you can invoke the `authenticated` method. You do not need to check `authenticated` with `Parse.User` objects that are obtained via an authenticated method.

## Encrypting Current User

> Often you want to be more careful with the user information stored in the browser, if this is your case try the encryption of the current user object. To enable this functionality, call `Parse.enableEncryptedUser()` and set your secret token.

Reviewer 2: ```suggestion
Often you may want to be more careful with user information stored in the browser, if this is the case you can encrypt the current user object:
```

```javascript

Parse.enableEncryptedUser();
Parse.secret = 'my Secrey Key';

```

> *   It´s important to reamember that this function is not gonna work if `Parse.secret` is not setted.

Reviewer 2: ```suggestion
*   It's important to remember that this function will not work if `Parse.secret` is not set.
```

> *   Also you need to remember that this works only in browser.

Reviewer 2: ```suggestion
*   Also note that this only works in the browser.
```

> With this your record in the Local Storage looks like a random string and only can be read from `Parse.User.current()`

Reviewer 2: ```suggestion
Now the record in Local Storage looks like a random string and only can be read using `Parse.User.current()`
```

> You can check if this feature is anable with the function `Parse.isEncryptedUserEnabled()`

Reviewer 2: ```suggestion
You can check if this feature is enabled with the function `Parse.isEncryptedUserEnabled()`.
```
